app/views/handlers/content.py

```diff
@@ -28,24 +29,25 @@ def rendered_block(self):
             list_item_id=self._current_location.list_item_id,
         )
 
-    def get_context(self):
+    def get_context(self) -> dict | None:
         return {
             "block": self.rendered_block,
             "individual_response_url": individual_response_url(
-                self._schema.get_individual_response_list(),
-                self._current_location.list_item_id,
+                # type ignore: if block is first in individual response, these two items wont be none
+                self._schema.get_individual_response_list(),  # type: ignore
+                self._current_location.list_item_id,  # type: ignore
                 self._questionnaire_store,
             )
             if self._is_block_first_block_in_individual_response()
             else None,
         }
 
-    def _get_content_title(self, transformed_block):
+    def _get_content_title(self, transformed_block: dict) -> str | None:
```

Comment:
the transform_block is passed in by the Content class which is using transformed_variants to get it - so it'll be an `ImmutableDict`